options.go

```diff
@@ -193,7 +195,6 @@ type Options struct {
 	Compression    int
 	Zoom           int
 	Crop           bool
-	SmartCrop      bool
```

Comment:
Reverted the SmartCrop change and validated that both parameters produce identical results.